webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -28,4 +48,146 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
     elif disability_degree >= 30:
         return 620
     elif disability_degree >= 20:
-        return 384
\ No newline at end of file
+        return 384
+    
+    return 0
```

Comment:
Could you add a test for that? Or - in other words: Why even return 0 instead of None?

Reply:
To display a plausible value instead of "None Euro" but maybe a error message would be better?

Reply:
No no, this is a valid case and we should not throw an error for it. Or do you mean when displaying the message? Yes, that should throw an error, I think.

Reply:
Hm thought if we need to display a error if the calculation returns a None or Zero... But maybe this is to technical for the user? Should we just log?

Reply:
If the calculation returns None or zero, then the person is just not eligibile to get a pauschbetrag, that's the semantic of 0 or None. So: as long as it's consistent (Does 0 or None mean "no pauschbetrag claim"?), I don't care.